source/tests/e2e/tests/transactions/Send.spec.js

```diff
@@ -0,0 +1,83 @@
+import assert from 'assert';
+
+import { beforeEach, afterEach } from 'mocha';
+import { buildWebDriver } from '../webdriver';
+import { importWallet } from '../initialize';
+import { By } from 'selenium-webdriver';
+import { storeState } from '../../../source/state/store';
+
+describe('Send screen tests', async () => {
+  let uiWebDriver = null;
+
+  beforeEach(async () => {
+    const { driver } = await buildWebDriver();
+
+    uiWebDriver = driver;
+
+    await driver.navigate();
+    await importWallet({ driver });
+  });
+
+  afterEach(() => {
+    uiWebDriver.quit();
+  });
+
+  it("should check if send form it's being shown", async () => {
+    const sendBalance = await uiWebDriver.findElement(By.id('send-balance'));
+
+    assert.ok(
+      typeof sendBalance === 'object',
+      '<!> Cannot find badge connected <!>'
+    );
+
+    const receiverInput = await uiWebDriver.findElement(
+      By.id('receiver-input')
+    );
+
+    assert.ok(
+      typeof receiverInput === 'object',
+      '<!> Cannot find receiver input <!>'
+    );
+
+    const networkDropdown = await uiWebDriver.findElement(
+      By.id('send-network-dropdown')
+    );
+
+    assert.ok(
+      typeof networkDropdown === 'object',
+      '<!> Cannot find network dropdown <!>'
+    );
+
+    const verifyAddressSwitch = await uiWebDriver.findElement(
+      By.id('verify-address-switch')
+    );
+
+    assert.ok(
+      typeof verifyAddressSwitch === 'object',
+      '<!> Cannot find verify address switch <!>'
+    );
+
+    const ZDagSwitch = await uiWebDriver.findElement(By.id('z-dag-switch'));
```

Comment:
Use lower camel case
```suggestion
    const zDagSwitch = await uiWebDriver.findElement(By.id('z-dag-switch'));
```